src/common/BaseAnnotator.ts

```diff
@@ -66,6 +66,7 @@ export default class BaseAnnotator {
         // Add custom handlers for events triggered by the Preview SDK
         this.addListener(LegacyEvent.SCALE, this.handleScale);
         this.addListener(Event.ACTIVE_SET, this.setActiveAnnotationId);
+        this.addListener(Event.ANNOTATION_REMOVE, this.deleteAnnotation);
```

Comment:
for consistency -- should all these methods, actions, etc... be renamed to `remove*` as well?